QModManager/ErrorMessage.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace QModManager
+{
+    public static class ErrorMessage
+    {
+        public static void ShowError(string error, Action<bool> function, string leftButtonText = "Yes", string rightButtonText = "No")
+        {
+            // Still need to implement leftButtonText and rightButtonText
+            uGUI_SceneConfirmation confirmation = uGUI.main.confirmation;
+
+            if (leftButtonText == "" || leftButtonText == null) confirmation.no.gameObject.SetActive(false);
```

Comment:
Use `string.IsNullOrEmpty(` when `string` values for null and empty.

Reply:
Addressed in commit 837a2bd